docs/quickstart.rst

```diff
@@ -61,12 +61,18 @@ For this example we use the
 objective is to predict whether a person makes more (label 1) or less (0)
 than $50,000 a year.
 
+.. doctest::
+
     >>> import numpy as np 
     >>> import matplotlib.pyplot as plt 
     >>> from shap.datasets import adult  # shap is only used its dataset utility
     >>> X, y_true = adult()
     >>> y_true = y_true * 1
     >>> sex = X['Sex'].apply(lambda sex: "female" if sex == 0 else "male")
+    >>> sex.value_counts()
+    male      21790
+    female    10771
+    Name: Sex, dtype: int64
 
 .. figure:: auto_examples/quickstart/images/sphx_glr_plot_adult_dataset_001.png
```

Comment:
I hadn't spotted that this was from an `auto_example` .... I think that would complicate things.

Reply:
I wonder whether there's a solution to this. I don't expect the dataset to change anytime soon, so I'm not super worried.

Reply:
I'll leave for now, then.